modules/configuration/src/main/java/org/apache/ignite/internal/configuration/util/ConfigurationUtil.java

```diff
@@ -1149,4 +1149,13 @@ public static <T extends ConfigurationProperty<?>> T getByInternalId(NamedConfig
     public static <N> N getByInternalId(NamedListView<N> node, UUID internalId) {
         return node.get(((NamedListNode<?>) node).keyByInternalId(internalId));
     }
+
+    /**
+     * Checks whether configuration schema contains {@link AbstractConfiguration}.
+     *
+     * @param schemaClass Configuration schema class.
+     */
+    public static boolean isAbstractConfiguration(Class<?> schemaClass) {
```

Comment:
Do we really need this method? It doesn't make the code shorter nor more readable

Reply:
Yes, it makes it shorter, if we remove it, then similar methods should also be removed, I suggest leaving it as it is wdyt?

Reply:
I would suggest removing this particular one, we can remove others during a separate refactoring

Reply:
It is better to deal with all in one ticket so that there is one style if needed.
https://issues.apache.org/jira/browse/IGNITE-17166

Reply:
but this is a new method that you introduce in this PR, what's the point of leaving it if you are going to remove it later?

Reply:
Fix it.